packages/components/select/src/select.tsx

```diff
@@ -29,8 +29,11 @@ function Select<T extends object>(props: Props<T>, ref: ForwardedRef<HTMLSelectE
     endContent,
     placeholder,
     renderValue,
+    isOutsideLeft,
```

Comment:
wt's that for?

Reply:
idk, maybe it appeared when I updated the branch. will check again